examples/field_cu/field_step.cu

```diff
@@ -0,0 +1,221 @@
+// SPDX-FileCopyrightText: 2020 CERN
+// SPDX-License-Identifier: Apache-2.0
+
+// Author: J. Apostolakis  12 Nov 2020
+
+#include <cstdio>
+#include <iomanip>
+
+#include <CopCore/SystemOfUnits.h>
+#include <CopCore/PhysicalConstants.h>
+
+// #include <CopCore/Ranluxpp.h>
+
+#include <VecGeom/base/Vector3D.h>
+#include "track.h"
+
+#include <AdePT/BlockData.h>
+
+#include "uniformMagField.h"
+
+#include "ConstFieldHelixStepper.h"
+
+
+#if 0
+#include "IterationStats.h"
+
+// Statistics for propagation chords
+IterationStats      *chordIterStatsPBz      = nullptr;
+__device__
+IterationStats_impl *chordIterStatsPBz_impl = nullptr;
+// Needed for fieldPropagatorConstBz.h -- for now
+#endif
```

Comment:
Is this still needed?

Reply:
Yes - as field_step is for testing the field I will be using it in future.